reinforcement_learning/rl_on_gcp_demo/trainer/ddpg_agent.py

```diff
@@ -0,0 +1,111 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation of a DDPG agent.
+
+Implementation of DDPG - Deep Deterministic Policy Gradient
+Algorithm and hyperparameter details can be found here:
+http://arxiv.org/pdf/1509.02971v2.pdf
+
+"""
+import agent
+from common import replay_buffer
+from common.actor_critic import ActorNetwork
+from common.actor_critic import CriticNetwork
+import numpy as np
+
+
+class DDPG(agent.Agent):
```

Comment:
How does the proposed `agent` module compare with the existing `tf-agents`?  https://github.com/tensorflow/agents/blob/master/tf_agents/agents/tf_agent.py

(`tf-agents` comes with also TD3 and DDPG implementations, and the other parts of the library might conform better to interfaces by deepmind and openai-gym.)

Reply:
please at least point out and comment on the existing tf-agents library in both the blog post and in the README of this set of samples if you don't have time to change the code or find a way to consolidate the different interfaces (your `agent` and `tf-agents`).